lua/bufferline/bbye.lua

```diff
@@ -144,13 +144,8 @@ function bbye.delete(action, force, buffer, mods)
     end
   end
 
-  -- Because tabbars and other appearing/disappearing windows change
-  -- the window numbers, find where we were manually:
-  for _, window in ipairs(window_numbers) do
-    if vim.w[window].bbye_back then
-      vim.api.nvim_command '1wincmd w'
-      vim.w.bbye_back = nil
-    end
```

Comment:
`window_numbers` could have changed between now and when the list was read. We avoid the issue by avoiding reusing a stale list.